include/RAJA/pattern/kernel/For.hpp

```diff
@@ -45,7 +46,7 @@ namespace statement
 
 /*!
  * A RAJA::kernel statement that implements a single loop.
- *
+ * Assigns the iterate to argument ArgumentId
```

Comment:
Minor nit for clarity: would this comment be better to say 'loop iterate' instead of just 'iterate'?